packages/navigation/index.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 import { Platform, StatusBar } from 'react-native';
-import ReactNavigation from 'react-navigation';
+import { StackNavigator as MainStackNavigator } from 'react-navigation'; // eslint-disable-line no-restricted-imports
```

Comment:
Why are you ignoring eslint rules here? I think it would be better to fix it instead.

Reply:
I agree in not ignoring the rule. Let's investigate it. Moreover, in `react-navigation 2` they use `createStackNavigator` instead.